Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -189,7 +188,7 @@ def consolidate_channel_torrent(self):
 
             return self.commit_channel_torrent(new_start_timestamp=start_timestamp)
 
-        def update_channel_torrent(self, metadata_list):
+        def update_channel_torrent(self, metadata_list, final_timestamp):
```

Comment:
* Please add the argument in the docstring

Reply:
Good point!